docker.py

```diff
@@ -33,12 +34,22 @@ def run():
                     err_fd.write(str(cmd) + "\n")
                     err_fd.flush()
                     print(cmd)
-                    subprocess.run(cmd, *args,
-                                   stdout=out_fd, stderr=err_fd, **kwargs)
-                    out_fd.write("\n")
-                    out_fd.flush()
-                    err_fd.write("\n")
-                    err_fd.flush()
+                    try:
+                        subprocess.run(cmd, *args,
+                                       stdout=out_fd, stderr=err_fd, **kwargs)
+                    except BaseException:
+                        formatted_exc = ("\nProcess failed:\n"
+                                         + traceback.format_exc())
+                        out_fd.write(formatted_exc + "\n")
+                        out_fd.flush()
+                        err_fd.write(formatted_exc + "\n")
+                        err_fd.flush()
+                        raise
+                    finally:
+                        out_fd.write("\n")
+                        out_fd.flush()
+                        err_fd.write("\n")
+                        err_fd.flush()
```

Comment:
Just out of interest, why has this changed?

Reply:
Reverted. Will go into a separate PR https://github.com/neo4j-drivers/testkit/pull/527